kubectl-plugin/component/component/logging.go

```diff
@@ -0,0 +1,46 @@
+package component
+
+import (
+	"github.com/kubesphere-sigs/ks/kubectl-plugin/common"
+)
+
+// Logging return the struct of Logging
+type Logging struct{}
+
+// GetName return the name of Logging
+func (l *Logging) GetName() string {
+	return "logging"
+}
+
+// Uninstall uninstall Logging
+func (l *Logging) Uninstall() error {
+	// To disable only log collection
+	if err := common.ExecCommand("kubectl", "delete", "inputs.logging.kubesphere.io", "-n", "kubesphere-logging-system", "tail"); err != nil {
+		return err
+	}
+
+	// To uninstall Logging system including Elasticsearch
+	if err := common.ExecCommand("kubectl", "delete", "crd", "fluentbitconfigs.logging.kubesphere.io"); err != nil {
```

Comment:
This same deletion cmds can use a for loop instead.

Reply:
You can use this style to delete:  `kubectl delete resourceType(crd or deployment or whatever)  resoucrceName(inputs.logging.kubesphere.io)  [addons -n kubesphere-logging-system ]` . And this part can bring a common function.